main.cpp

```diff
@@ -59,85 +59,86 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:         dog
+//  action 1:     bark
+dog.bark();
+//  action 2:     run
+dog.run();
+//  action 3:     sleep
+dog.sleep(); 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:         ball
+//  action 1:     bounce
+ball.bounce();
+//  action 2:     stop
+ball.stop();
```

Comment:
`stop` isn't really an action unless you were previously doing something and that's what you're stopping. 
- [ ] revise